src/View/Admin/Home/index.html.twig

```diff
@@ -1,4 +1,37 @@
 {% extends 'adminLayout.html.twig' %}
+{% block title %}
+	{{ parent() }}
+	- Accueil admin
+{% endblock %}
 {% block content %}
+	<main class='conteneur-fluid'>
```

Comment:
conteneur ?
![kanye-west-are-you-serious-right-now](https://user-images.githubusercontent.com/79161810/167607116-f644518a-f03d-4583-ad45-b83f18dfd200.gif)